Upload guide (#1000)
kartoza/miniSASS: docs/src/user/guide/upload.md

# Upload Data User Guide

Welcome to the `Add Record` user guide. This guide will walk you through the process of adding a new record using our user-friendly form. Follow the steps below to efficiently input data and contribute valuable information to our system.

## How can I access the add record form?

To access the `Add Record` form click on the 1️⃣ `Map` option available in the navigation bar.

![map option](./img/upload-1.png)

After clicking on this button, you will be redirected to the map page where you will find the 1️⃣ `Add Record` button, click on this button to open add record form.

![add record button](./img/upload-2.png)

> Upon clicking on this button the add record form will open. This form have four sections

Reviewer: ```suggestion
Upon clicking on this button the add record form will open. This form has four sections
```

## Site details section

For the site details you can create a new site or can select an existing site. To create a new site choose the 1️⃣ `Create New Site` option or to use the existing site choose the 2️⃣ `Use Existing Site` option.

![site details](./img/upload-3.png)

### Create New Site option

> By default the create new site option is selected. Fill the 1️⃣ required filds and upload the image of the site. To upload the image of the site click on the 2️⃣ `Upload site images` button.

Reviewer: ```suggestion
By default the create new site option is selected. Fill the 1️⃣ required fields and upload the image of the site. To upload the image of the site click on the 2️⃣ `Upload site images` button.
```

![create new site](./img/upload-4.png)

Upon clicking the popup will open, where you can select the images from the system. Click on the 1️⃣ `Browse` button to select the images from the system and then click on the 2️⃣ `upload chosen file` button to upload the images.

![upload images](./img/upload-5.png)

### Use Existing Site option

> Select this option by clickng on the 1️⃣ `Use Existing Site` radio button.

Reviewer: ```suggestion
Select this option by clicking on the 1️⃣ `Use Existing Site` radio button.
```

![use existing site](./img/upload-7.png)

Upon clicking the `Upload site images` button, a popup will open, where you can select the images from the system. Click on the 1️⃣ `Browse` button to select the images from the system and then click on the 2️⃣ `upload chosen file` button to upload the images.

![upload images](./img/upload-5.png)

After clicking the `Select site on map` button, select site location on the map. Upon selecting the site on the map all the fields will be filled.

![use existing site](./img/upload-8.png)

## Site Location section

>Note: If you choose site from the map the `Site Location` section will be disabled.

There are two option to enter the site location  information, either by clicking on the 1️⃣ `Site on map` button or by clicking on the 2️⃣ `Type in coordinates` button.

![site location](./img/upload-9.png)

### Select on map option

After clicking the `Select on map` button, select site location on the map.

![select on map](./img/upload-10.png)

Upon selecting the site on the map latitude and longitude will be filled automatically.

![select on map](./img/upload-11.png)

### Type in coordinates option

> In this option you can enter the coordinates manually via two different types. You can either enter DMS coorrdinates or degree coordinate.

Reviewer: ```suggestion
In this option you can enter the coordinates manually via two different types. You can either enter DMS coordinates or degree coordinate.
```

By default the degree coordinates (latitude and longitude) is selected.

![degree coordinates](./img/upload-12.png)

> To enter the DMS coordinates click on the redio button available in front of the DMS.

Reviewer: ```suggestion
To enter the DMS coordinates click on the radio button available in front of the DMS.
```

![DMS coordinates](./img/upload-13.png)

## Observation details section

> Enter the collectors name and the notes in the 1️⃣ available fields and select the 2️⃣ date from the calander.

Reviewer: ```suggestion
Enter the collectors name and the notes in the 1️⃣ available fields and select the 2️⃣ date from the calendar.
```

> ![observation ditails](./img/upload-14.png)

Reviewer: ```suggestion
![observation details](./img/upload-14.png)
```

To select the date from the calendar click on the calendar icon available in the date field.

![calendar](./img/upload-15.png)

## Measurements section

> Enter the measurment details in the 1️⃣ available fields. and the click on the 

Reviewer: ```suggestion
Enter the measurement details in the 1️⃣ available fields and the click on the next button
```

> ![measurment](./img/upload-16.png)

Reviewer: ```suggestion
![measurement](./img/upload-16.png)
```

> >Note: If the `Next` button is not clickable and the incomplete form message (Incomplete form. Please scroll up to complete the highlighted fields.) shown on the form that means you have forgotten to fill the required field.

Reviewer: ```suggestion
>Note: If the `Next` button is not clickable and the incomplete form message (Incomplete form. Please scroll up to complete the highlighted fields.) is shown on the form that means you have forgotten to fill the required fields.
```

> Fill all the required fields to proceed further. After filling all the required fields the incomplete form message will not show on the page and the `Next` button will be clickable. Click on the 1️⃣ next button for next step of data uploading.

Reviewer: ```suggestion
Fill all of the required fields to proceed further. After filling all the required fields the incomplete form message will not show on the page and the `Next` button will be clickable. Click on the 1️⃣ next button for next step of data uploading.
```

> ![measurment](./img/upload-17.png)

Reviewer: ```suggestion
![measurement](./img/upload-17.png)
```

> Upon clicking the next button the score form will open. Where you can select the group by clicking on the checkbox available in front of the group name and upload the images by clickin on the `Upload images` button available after the name of the group.

Reviewer: ```suggestion
Upon clicking the next button the score form will open. Where you can select the group by clicking on the checkbox available in front of the group name and upload the images by clicking on the `Upload images` button available after the name of the group.
```
